lib/core/include/irods/irods_server_properties.hpp

```diff
@@ -282,43 +282,6 @@ namespace irods
     {
         return irods::get_server_property<T>(configuration_parser::key_path_t{KW_CFG_ADVANCED_SETTINGS, _prop});
     } // get_advanced_setting
-
-    /// Returns the amount of shared memory that should be allocated for the DNS cache.
-    ///
-    /// \return An integer representing the size in bytes.
-    /// \retval 5000000          If an error occurred or the size was less than or equal to zero.
-    /// \retval Configured-Value Otherwise.
-    ///
-    /// \since 4.2.9
-    auto get_dns_cache_shared_memory_size() noexcept -> int;
```

Comment:
I'm guessing we don't need to deprecate these because these are pretty deep in the server despite being in `lib`?

Reply:
The plan is to mark them deprecated in 4.3.5. An issue for removing these in 5.0.0 would appear as well. I delayed creating the issues to make sure the work could be completed on time.

Reply:
Opened #8417 and #8418.